packages/cadl-ranch-specs/http/models/property-types/main.cadl

```diff
@@ -6,6 +6,8 @@ using Cadl.Rest;
 using Cadl.Http;
 using Azure.Core.Foundations;
 
+@Cadl.Rest.produces("application/json")
+@Cadl.Rest.consumes("application/json")
```

Comment:
I assume they are going to be deprecated anyway?

Reply:
It would be a long way I think. Currently, we .net still use these decorators.

Reply:
We shouldn’t add this. As @weidongxu-microsoft  it will most likely get deprecated on the next design meeting and even if it wasn’t the default is assumed to be json. So either way the csharp generator shouldn’t be needing it to function.

Reply:
We really need this. Currenly we need these to be the default value of content-type and accept, otherwise no headers will be generated. 

cc @lirenhe the information seems not aligned with ours. We cannot generate workable SDKs without these two lines.

Reply:
this is a problem with the csharp generator... 

This is a spec for what the generator should do. **It is valid.** It is the entire point of this repo, defining the behavior the generators should be respecting.

The generator should be fixed.

Reply:
Removed.